models/staging/stg_contract_data.yml

```diff
@@ -0,0 +1,153 @@
+version: 2
+
+models:
+  - name: stg_contract_data
+    tests:
+      - dbt_utils.recency:
+          datepart: hour
+          field: cast(closed_at as timestamp)
+          interval: '{{ 1 if target.name == "prod" else 6 }}'
+          meta:
+            description: "Monitors the freshness of your table over time, as the expected time between data updates."
+      - incremental_unique_combination_of_columns:
+          date_column_name: "closed_at"
+          greater_than_equal_to: "2 day"
+          combination_of_columns:
+            - ledger_key_hash
+            - last_modified_ledger
+            - ledger_entry_change
+          meta:
+            description: "Tests the uniqueness combination of: ledger_key_hash, last_modified_ledger, and ledger_entry_change."
+    columns:
+      - name: contract_id
+        description: '{{ doc("contract_id") }}'
+        tests:
+          - incremental_not_null:
+              date_column_name: "closed_at"
+              greater_than_equal_to: "2 day"
+
+      - name: contract_key_type
+        description: '{{ doc("contract_key_type") }}'
+        tests:
+          - incremental_not_null:
+              date_column_name: "closed_at"
+              greater_than_equal_to: "2 day"
+
+      - name: contract_durability
+        description: '{{ doc("contract_durability") }}'
+        tests:
+          - incremental_not_null:
+              date_column_name: "closed_at"
+              greater_than_equal_to: "2 day"
+
+      - name: asset_code
+        description: '{{ doc("asset_code") }}'
+
+      - name: asset_issuer
+        description: '{{ doc("asset_issuer") }}'
+
+      - name: asset_type
+        description: '{{ doc("asset_type") }}'
+        tests:
+          - incremental_not_null:
+              date_column_name: "closed_at"
+              greater_than_equal_to: "2 day"
+          - incremental_accepted_values:
+              date_column_name: "closed_at"
+              greater_than_equal_to: "2 day"
+              values: ["credit_alphanum4", "credit_alphanum12", "native"]
+              quote: true
```

Comment:
Need to remove this test for now. Context in https://stellarorg.atlassian.net/browse/HUBBLE-574